google/cloud/bigtable/internal/bigtable_stub_factory_test.cc

```diff
@@ -41,11 +42,15 @@ using ::google::cloud::testing_util::StatusIs;
 using ::google::cloud::testing_util::ValidateMetadataFixture;
 using ::testing::Contains;
 using ::testing::HasSubstr;
-using ::testing::IsEmpty;
-using ::testing::Not;
 using ::testing::NotNull;
+using ::testing::Pair;
 using ::testing::Return;
 
+MATCHER(IsWebSafeBase64, "") {
+  std::regex regex(R"re([A-Z0-9_-]+)re");
```

Comment:
Is the empty string WebSafeBase64?  If so, you may prefer to keep `IsWebSafeBase64()` and `Not(IsEmpty())` orthogonal.

Reply:
Done.